package.json

```diff
@@ -46,6 +46,7 @@
     "publish": "npm run build && cd build && npm publish --tag next"
   },
   "peerDependencies": {
+    "prop-types": "^15.0.0",
```

Comment:
`prop-types` won't conflict. Adding it to peer dependencies will further complicate installs. Many won't use prop-types. Like some typescript users (me included :grin:). Just add it to normal dependencies. fewer missing peer dependency warns and fewer breakage to take care of.